browser/src/control/jsdialog/Widget.TreeView.ts

```diff
@@ -1620,6 +1620,21 @@ class TreeViewControl {
 		if (level === 1 && !hasSelectedEntry) this.makeTreeViewFocusable(true);
 	}
 
+	showSearchBar(parent: HTMLElement) {
+		const searchBox = document.createElement('input');
+		searchBox.id = 'ui-treeview-search-input'; // Form fields should have either a name or an ID - using this instead of a class
```

Comment:
I can imagine we could have more of these at the same time.
For a11y we need to be sure it has unique id -> https://github.com/CollaboraOnline/online/blob/2b56dee33a59dd8c3a6f0add6775b9391ccaf32f/browser/src/control/jsdialog/Util.UniqueId.ts#L20